src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/common/model/in/ExceptionRecord.java

```diff
@@ -29,7 +33,11 @@ public ExceptionRecord(
         @JsonProperty("delivery_date") LocalDateTime deliveryDate,
         @JsonProperty("opening_date") LocalDateTime openingDate,
         @JsonProperty("scanned_documents") List<InputScannedDoc> scannedDocuments,
-        @JsonProperty("ocr_data_fields") List<OcrDataField> ocrDataFields
+        @JsonProperty("ocr_data_fields") List<OcrDataField> ocrDataFields,
+        @JsonProperty("envelope_id") String envelopeId,
+        @JsonProperty("is_automated_process") String isAutomatedProcess,
```

Comment:
may be we can name class field  differently automatedProcessFlag as it is not boolean ?

Reply:
I changed the field type as `boolean`